src/plugins/dashboard/public/dashboard_container/embeddable/create/create_dashboard.test.ts

```diff
@@ -532,7 +531,6 @@ test('creates a control group from the control group factory and waits for it to
     undefined,
     { lastSavedInput: expect.objectContaining({ controlStyle: 'oneLine' }) }
   );
-  expect(mockControlGroupContainer.untilInitialized).toHaveBeenCalled();
```

Comment:
I wanted to remove the `untilInitialized` function all-together since Dashboard is the only one that uses it, but some tests rely on it and I wanted to get this fix in ASAP 🙈 This will be cleaned up in the embeddable refactor, though.